src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -7246,7 +7246,7 @@ public void M1()
 parseOptions: TestOptions.Regular);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/15508"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
```

Comment:
>Skip = "https://github.com/dotnet/roslyn/issues/15508" [](start = 14, length = 54)

It is fine to merge with skipped test into a feature branch, but I think we should come up with solution or implement a workaround that prevents the crash before we merge this to master.